prow/config.yaml

```diff
@@ -12095,451 +12095,451 @@ periodics:
   interval: 1h
   agent: kubernetes
   labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
+    preset-service-account: 'true'
+    preset-k8s-ssh: 'true'
   name: ci-kubernetes-e2e-gke-cos-k8sbeta-default
   spec:
     containers:
     - args:
       - --bare
       - --timeout=70
       env:
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20180511-51cf02e75-master
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20180605-65a672b59-master
 - tags:
   - generated   # AUTO-GENERATED by experiment/generate_tests.py - DO NOT EDIT!
   interval: 1h
   agent: kubernetes
   labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
+    preset-service-account: 'true'
+    preset-k8s-ssh: 'true'
   name: ci-kubernetes-e2e-gke-cos-k8sbeta-ingress
   spec:
     containers:
     - args:
       - --bare
       - --timeout=170
       env:
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20180511-51cf02e75-master
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20180605-65a672b59-master
 - tags:
   - generated   # AUTO-GENERATED by experiment/generate_tests.py - DO NOT EDIT!
   interval: 1h
   agent: kubernetes
   labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
+    preset-service-account: 'true'
+    preset-k8s-ssh: 'true'
```

Comment:
/shrug

I didn't find an option for use single/double quotes..

Reply:
this is auto generated AFAIK 😱 
seems to work though?

Reply:
I mean it's probably from one of the yaml.dump

this is fine?

Reply:
http://yaml.org/spec/1.2/spec.html#id2788097
single quotes are definitely valid

Reply:
I mean if we should unify everything to single quote..